install_config/aggregate_logging.adoc

```diff
@@ -427,15 +427,18 @@ must be at least 512M. Possible suffixes are G,g,M,m.
 |A key/value map added to a PVC in order to select specific PVs.
 
 |`openshift_logging_es_pvc_dynamic`
-|Set to `true` to have PVC claims annotated so that their backing storage is
-dynamically provisioned. When set to `true` the storage class is ignored and not
-set to the PVC.
+|Set to `true` so the backing storage is dynamically provisioned. When set to
+`true` the storageClass spec is omitted from the PVC definition. The setting of
+`openshift_logging_es_pvc_storage_class_name` can override
+`openshift_logging_es_pvc_dynamic`.
 
 |`openshift_logging_es_pvc_storage_class_name`
 |To use a non-default storage class, set the variable with the storage class
 name. For example, set to one of the following,
```

Comment:
I'd say, "name, such as \`glusterprovisioner\` or \`cephrbdprovisioner\`. After you specify the storage class name, dynamic volume provisioning is active regardless of the  `openshift_logging_es_pvc_dynamic` value."